packages/@lwc/integration-not-karma/configs/base.mjs

```diff
@@ -20,32 +21,19 @@ const env = {
     LWC_VERSION,
     NODE_ENV: options.NODE_ENV_FOR_TEST,
 };
+
 /** @type {import("@web/test-runner").TestRunnerConfig} */
 export default {
-    files: [
-        // FIXME: These tests are just symlinks to integration-karma for now so the git diff smaller
-        'test-hydration/**/*.spec.js',
-        '!test-hydration/light-dom/scoped-styles/replace-scoped-styles-with-dynamic-templates/index.spec.js',
-    ],
     nodeResolve: true,
-    rootDir: import.meta.dirname,
+    rootDir: join(import.meta.dirname, '..'),
     plugins: [
         {
             resolveImport({ source }) {
                 if (source === 'test-utils') {
-                    return '/helpers/wtr-utils.mjs';
+                    return '/helpers/utils.mjs';
                 } else if (source === 'wire-service') {
-                    return '@lwc/wire-service';
-                }
-            },
-            async serve(ctx) {
```

Comment:
`serve` is moved to a separate plugin file, since it is different for regular tests vs hydration tests.